v-next/hardhat/src/internal/utils/package.ts

```diff
@@ -17,3 +18,27 @@ export async function getHardhatVersion(): Promise<string> {
 
   return packageJson.version;
 }
+
+export async function getLatestHardhatVersion(): Promise<string> {
```

Comment:
I think we should remove it after all and use the current hardhat version instead. By doing that we can add proper testing to the project templates (as they are coupled with the initialization process).